app/views/root/index.html.erb

```diff
@@ -0,0 +1 @@
+<div id="root">
```

Comment:
- [ ] There is an error coming from app/views/layouts/application.html.erb when loading the index page only, it is complaining that application.css does not exist, page works well when removing it, so please check this
![image](https://user-images.githubusercontent.com/62705947/223469961-e19171e1-5e92-4e1c-b661-230e0dfc0d20.png)

Reply:
@GzimAsani the project is working fine with me.
![Capture](https://user-images.githubusercontent.com/49188940/223472761-b9d4494e-281f-4aef-8aea-c6d613e6e60b.PNG)

Reply:
- Hi @myaserkhan, ignore this request than, it must've been something wrong with my PC